cde-core/src/pt/webdetails/cdf/dd/model/inst/writer/cdfrunjs/dashboard/amd/PentahoCdfRunJsDashboardWriteContext.java

```diff
@@ -50,5 +55,16 @@ public String replaceTokens( String content ) {
       //build the system resource links
       .replaceAll( ABS_SYS_RES_TAG, "/" + getSystemDir() + "/" + getPluginId( path ) + "$1" )
       .replaceAll( REL_SYS_RES_TAG, path + "$1" );
+
+    return result;
+  }
+
+  // method is "protected" in order to being accessible from tests
```

Comment:
Please remove this comment after moving this method to the Class from which this extends (see comment below).